File: BlackBerry/AlertMsgs/WriteClipboardCmd.java
package com.mlhsoftware.Blurts;

import org.json.me.*;


class WriteClipboardCmd extends CmdBase 
{
  // Tag Names
  private static final String KEY_TEXT = "Text";


  public WriteClipboardCmd( String string ) throws JSONException
  {
    super( string );
  }

  public WriteClipboardCmd()
  {
    super( CMD_WRITECLIPBOARD );
  }

  String getText()
  {
    return optString( KEY_TEXT );
  }
}

File: Desktop/Domain/CmdMsgs/WriteClipboardCmd.cs
using System;

using System.Xml;
using System.Xml.Xsl;
using System.Collections;
using System.Runtime.InteropServices;

namespace Blurts
{
  [ComVisible(true)]
  public class WriteClipboardCmd : CmdBase
  {
    // Tag Names
    private static String KEY_TEXT = "Text";


    public WriteClipboardCmd(String s)
      : base(s)
    {
    }

    public WriteClipboardCmd()
      : base(CMD_WRITECLIPBOARD)
    {
    }

    [ComVisible(true)]
    public String Text
    {
      get
      {
        return optString(KEY_TEXT);
      }
      set
      {
        try
        {
          remove(KEY_TEXT);
          put(KEY_TEXT, value);
        }
        catch (JSONException e)
        {
          Console.WriteLine(e.ToString());
        }
      }
    }
  }
}
